### PR TITLE
Adds ability to import sitemaps to include a website

### PR DESCRIPTION
**Pull request by skidvis**
Adds sitemap.py, a new menu link in main.py for sitemaps, and refactors link.py to reuse its link parsing in sitemap.y

Allows one to provide a link to a sitemap url as a list of links to include.

### CODE REVIEW

File: collector/scripts/sitemap.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    root = ET.fromstring(response.content)
+    
+    urls = []
+    for element in root.iter('{http://www.sitemaps.org/schemas/sitemap/0.9}url'):
+        for loc in element.iter('{http://www.sitemaps.org/schemas/sitemap/0.9}loc'):
+            urls.append(loc.text)
+    
+    return urls
+
+def sitemap():
```
Comment (timothycarambat) [inline collector/scripts/sitemap.py:16]: Can you add a comment above this line like there is in `links.py` to a URL of a sitemap (can be anywhere) as it helps people know what a good example target looks like.

Comment (timothycarambat) [reply collector/scripts/sitemap.py:16]: Decent example: `https://www.nerdwallet.com/blog/wp-sitemap-news-articles-1.xml`

Otherwise, looks good to merge

**Review by timothycarambat — CHANGES_REQUESTED** [review]
Just a comment of an example sitemap link above the executing method would help others know what kind of data is acceptable. Works beautifully though 🎉

**Comment by skidvis** [comment]
I have added the comment with the example sitemap as requested.

**Review by timothycarambat — APPROVED** [review]
LGTM!

**Pull request merged**